Intersect.Editor/Forms/Editors/Events/CommandPrinter.cs

```diff
@@ -925,9 +925,17 @@ private static string GetCommandText(WarpCommand command, MapInstance map)
                 }
             }
 
-            return Strings.EventCommandList.warp.ToString(
-                mapName, command.X, command.Y, Strings.Directions.dir[(int) command.Direction - 1]
-            );
+            if (command.ChangeInstance)
+            {
+                return Strings.EventCommandList.InstancedWarp.ToString(
+                    mapName, command.X, command.Y, Strings.Directions.dir[(int)command.Direction - 1], Enum.GetName(typeof(MapInstanceType), command.InstanceType)
```

Comment:
If `command.InstanceType` is MapInstanceType instead of int, just using `ToString()` will work.

Reply:
Done